01/Table.js

```diff
@@ -1,12 +1,25 @@
-import React from 'react';
+import React from "react";
+import TableHeader from "./TableHeader";
+import TableBody from "./TableBody";
+import TableRow from "./TableRow";
+import TableFooter from "./TableFooter";
 
 class Table extends React.Component {
-    render() {
-        const {data} = this.props;
-        console.log(data);
+  render() {
+    const { data } = this.props;
 
-        return <table></table>
-    }
+    return (
+      <table>
+        <TableHeader data={data[0]} />
+        <TableBody>
+          {data.map((item) => {
+            return <TableRow key={item.id} data={item} />;
+          })}
```

Comment:
Można by było też napisać metodę w klasie o nazwie `renderTableRows()` tutaj ją wywołać, a w niej wykonać `.map()` - ładniej by to wyglądało (czytelniej)